packages/svelte/tests/migrate/samples/$$slots-used-as-variable-$$props/output.svelte

```diff
@@ -1,5 +1,11 @@
 <script>
-	/** @type {{message?: import('svelte').Snippet, extra?: import('svelte').Snippet<[any]>, [key: string]: any}} */
+	/**
+	* @typedef {Object} Props
```

Comment:
could we insert a space between the tab and `*` on this and the next three lines?

Reply:
Uhhh really good eye...will do